### PR TITLE
Bug fixes and enchancements in integration

### PR DESCRIPTION
**Pull request by cyble-dev**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by RosenbergYehuda** [comment]
Hello @cyble-dev, thank you for your resubmission.As you can see in the GitHub actions, in the "ci/circleci: Run Validations" section, that there are many validation failures.Do you know how to use our demisto-sdk format?If yes, please try that and see if the errors are gone.Please check after that that everything is in the right place and working.

**Comment by cyble-dev** [comment]
Hi @RosenbergYehuda,
As per your suggestion, I checked the failing test.
Before creating the PR, I have validated the files with the command:

`demisto-sdk format -ud -i /Users/<user_name>/Desktop/content/Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.yml`

File: Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+                            "right": {
+                                "isContext": false,
+                                "value": {
+                                    "simple": "compromised_cards"
+                                }
+                            },
+                            "type": "shortText"
+                        }
+                    ]
+                ],
```
Comment (cyble-dev) [inline Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json:459]: Hi @RosenbergYehuda,
As per your suggestion, I have updated all the files and fixed all the issues that were coming in `ci/circleci: Run Validations` section

However, I am still getting this error

```
[2023-10-18 15:59:47,043] [ERROR] Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json: [ST111] - The field "filters" in path 'quickView'-> 'sections'-> 'Compromised Card Details' was not defined in the scheme
[2023-10-18 15:59:47,043] [INFO] To fix the problem, try running `demisto-sdk format -i Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json`
```

I was able to fix this too, but I had to remove the `filter` key from the layout as highlighted:
If I remove this part, then the test is passing.

But I believe this filter is required to check if the coming incident is of this particular type: 'compromised_card'. So removing this filter does not make sense. But keeping it is not letting the test pass.
I have exported this layout json from XSOAR instance.

Please suggest how we can go about this.

Comment (RosenbergYehuda) [reply Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json:459]: Good job solving most of validations!
@melamedbn , can you please assist here?

**Comment by melamedbn** [comment]
Hi @cyble-dev,

Everything looks great, just one fix needed:
Please remove the hardcoded values from the playbook inputs.

Best regards,
Ben

**Comment by cyble-dev** [comment]
Hi @melamedbn,
Can you please mention which values are you referring to?

**Comment by melamedbn** [comment]
> Hi @melamedbn, Can you please mention which values are you referring to?

The dates in the playbook inputs.

File: Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+                "name": "Incident Information",
+                "query": null,
+                "queryType": "",
+                "readOnly": false,
+                "type": ""
+            }
+        ]
+    },
+    "group": "incident",
+    "id": "CybleEventsv2-Layout",
```
Comment (melamedbn) [inline Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json:369]: Please remove the suffix -Layout

File: Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+    "name": "CybleEventsv2-Layout",
```
Comment (melamedbn) [inline Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json:370]: Please remove the suffix -Layout

File: Packs/CybleEventsV2/Playbooks/playbook-Cyble_Vision_Alert_V2.yml [hunk 4]
```diff
@@ -91,13 +124,13 @@ inputs:
     simple: "2023-05-30T00:00:00+00:00"
   required: true
   description: ""
-  playbookInputQuery: null
+  playbookInputQuery:
 - key: end_date
   value:
     simple: "2023-05-31T00:00:00+00:00"
```
Comment (cyble-dev) [inline Packs/CybleEventsV2/Playbooks/playbook-Cyble_Vision_Alert_V2.yml:130]: Hi @melamedbn,
Are you talking about these dates? If yes, then I am not sure what to put here?
Does this section have same inputs as in line 53 and 61 for start and end dates?

Comment (cyble-dev) [reply Packs/CybleEventsV2/Playbooks/playbook-Cyble_Vision_Alert_V2.yml:130]: Hi @melamedbn,
Can we please escalate this? Let me know the expected edit in above lines 124 & 130.

**Comment by cyble-dev** [comment]
Hi @melamedbn, @RosenbergYehuda,
I have made all the changes requested by you and a minor addition in the integration script.
Please review the PR and if possible, can we push it to the marketplace by today? Because I suppose tomorrow is your holiday.

**Comment by cyble-dev** [comment]
Hey @melamedbn, @RosenbergYehuda & @edik24,
I have made all the changes and fixed all the issues. The PR is passing all the tests as well.
Please take a look at the PR and please push it for approval so we can get the app live as soon as possible.
Thanks.

**Comment by melamedbn** [comment]
Hi @cyble-dev,

There are still things that should be addressed as discussed in our meeting.

1. Remove the hardcoded values from the playbook inputs.
2. Add description to the inputs.
3. Add another input for the 'limit' argument so the user will be able to choose the limit.

Best regards,
Ben

**Comment by cyble-dev** [comment]
Hi @melamedbn,
Can we get on a quick call right now?

**Comment by cyble-dev** [comment]
Hi @melamedbn,
I have made the changes you suggested. Please proceed.

**Comment by cyble-dev** [comment]
Hi @edik24, @melamedbn & @RosenbergYehuda,
I have removed the playbook’s changes and updated the whole pack. Please proceed.

Please let me know if you have any query.

**Comment by cyble-dev** [comment]
Hi @edik24, @melamedbn & @RosenbergYehuda,
I have removed the playbook’s changes and updated the whole pack. Please proceed.

Please let me know if you have any query.

File: Packs/CybleEventsV2/doc_files/Cyble_Vision_Alert_V2.png
Comment (melamedbn) [inline Packs/CybleEventsV2/doc_files/Cyble_Vision_Alert_V2.png:1]: Please remove the image as well

**Review by melamedbn — APPROVED** [review]

**Comment by cyble-dev** [comment]
> @cyble-dev,
> 
> Just saw that the current image isn't good. so let's keep the new image and delete the old one.

oh, okay.
So no modification is required, right?
We are moving forward as it is now.

**Review by RosenbergYehuda — APPROVED** [review]

**Pull request merged**